snapshot/policy/retention_policy.go

```diff
@@ -115,7 +118,7 @@ func (r *RetentionPolicy) ComputeRetentionReasons(manifests []*snapshot.Manifest
 }
 
 func (r *RetentionPolicy) effectiveKeepLatest() *OptionalInt {
-	if r.KeepLatest.OrDefault(0)+r.KeepHourly.OrDefault(0)+r.KeepDaily.OrDefault(0)+r.KeepWeekly.OrDefault(0)+r.KeepMonthly.OrDefault(0)+r.KeepAnnual.OrDefault(0) == 0 {
+	if r.KeepLatest.OrDefault(0)+r.KeepHourly.OrDefault(0)+r.KeepDaily.OrDefault(0)+r.KeepWeekly.OrDefault(0)+r.KeepMonthly.OrDefault(0)+r.KeepAnnual.OrDefault(0)+r.MinRetentionDays.OrDefault(0) == 0 {
```

Comment:
this is not correct, the value returns the "count of snapshots" not the "number of snapshot + number of days" which does not really make sense. I think reverting this line is fine.

Reply:
The problem is, if we don't add `r.MinRetentionDays.OrDefault(0)` and only `Min. Retention days` is set to some value and  other polices is set to zero then effectiveKeepLatest() function returns `keep-latest = maxInt`, so then every snapshot will have retention reasons as `[min-retention-days, latest-n ]` where `n = number of snapshots`

Reply:
@jkowalski Can you suggest something